Lib/fontTools/otlLib/builder.py

```diff
@@ -522,7 +1075,20 @@ def buildValue(value):
 # GDEF
 
 def buildAttachList(attachPoints, glyphMap):
-    """{"glyphName": [4, 23]} --> otTables.AttachList, or None"""
+    """Builds an AttachList subtable.
+
+    A GDEF table may contain an Attachment Point List table (``AttachList``)
```

Comment:
The formatting of table names is inconsistent throughout the documentation: sometimes it is like “AttachList” and sometimes like “`AttachList`”. FWIW the OpenType spec formats them like “AttachList”.